### PR TITLE
A11y: Move Placeholder Value to Label

### PR DESCRIPTION
**Pull request by andrewyu-salesforce**
Display label in the correct place instead of using placeholder so user can always see the description of the input field

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by andrewyu-salesforce** [comment]
Attach to [WI-795453](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008U821IAC/view)

**Comment by andrewyu-salesforce** [comment]
@RadSFDeveloper @npsp-reedestockton The text label PR is ready for review.

File: src/aura/autocomplete/autocomplete.cmp [hunk 1]
```diff
@@ -41,9 +41,9 @@
              data-typeahead="true">
             <div class="slds-form-element">
                 <aura:if isTrue="{!!empty(v.label)}">
-                    <label class="slds-form-element__label">{!v.label}</label>
+                    <span class="slds-form-element__label slds-float_left" id="inputLabel" >{!v.label}</span>
                 </aura:if>
-                <div class="slds-form-element__control slds-input-has-icon slds-input-has-icon_right" onfocusin="{!c.handleFocusInput}" tabindex="0">
+                <div class="slds-form-element__control slds-input-has-icon slds-input-has-icon_right" onfocusin="{!c.handleFocusInput}" aria-labelledby="inputLabel">
```
Comment (npsp-reedestockton) [inline src/aura/autocomplete/autocomplete.cmp:46]: What's the reason for removing tabindex="0"?

File: src/aura/autocomplete/autocomplete.cmp [hunk 2]
```diff
@@ -41,9 +41,9 @@
              data-typeahead="true">
             <div class="slds-form-element">
                 <aura:if isTrue="{!!empty(v.label)}">
-                    <label class="slds-form-element__label">{!v.label}</label>
+                    <span class="slds-form-element__label slds-float_left" id="inputLabel" >{!v.label}</span>
```
Comment (npsp-reedestockton) [inline src/aura/autocomplete/autocomplete.cmp:44]: Why span instead of label?

File: src/aura/autocomplete/autocomplete.cmp (hunk 1 above)
Comment (andrewyu-salesforce) [reply src/aura/autocomplete/autocomplete.cmp:46]: This tabindex makes the focus outside the input div, user needs to do 1 extra tabbing to get to the input focus.

File: src/aura/autocomplete/autocomplete.cmp (hunk 2 above)
Comment (andrewyu-salesforce) [reply src/aura/autocomplete/autocomplete.cmp:44]: <span> is recommended by the SLDS accessibility read only label. See [here](https://www.lightningdesignsystem.com/components/input/)

**Review by npsp-reedestockton — APPROVED** [review]
Great!  I learned a few things from this.  Thanks!

**Review by RadSFDeveloper — APPROVED** [review]
Besides the question about spacing between the page subtitle and household members, it looks good.

**Pull request merged**